src/ProgressOnderwijsUtils/ProgressOnderwijsUtils.csproj

```diff
@@ -2,11 +2,13 @@
   <Import Project="..\NugetPackagesCommon.props" />
   <PropertyGroup Label="Configuration">
     <Version>50.0.1</Version>
-    <PackageReleaseNotes>Represent parsed html &lt;template&gt; element contents in the resulting HtmlFragment</PackageReleaseNotes>
+    <PackageReleaseNotes>Added Uri extensions for safely combining paths.</PackageReleaseNotes>
     <Title>ProgressOnderwijsUtils</Title>
     <Description>Collection of utilities developed by ProgressOnderwijs</Description>
     <PackageTags>ProgressOnderwijs</PackageTags>
     <TargetFrameworks>netstandard2.0;netcoreapp3.0</TargetFrameworks>
+    <AssemblyVersion>50.1.0.0</AssemblyVersion>
+    <FileVersion>50.1.0.0</FileVersion>

```

Comment:
I think you just want to change `<Version>`

Reply:
Dit zal wel in de VS UI gebeurt zijn?

Reply:
My mistake.